cpu/cc2538/Makefile.features

```diff
@@ -0,0 +1 @@
+FEATURES_PROVIDED += periph_pm
```

Comment:
NO its *one* _essential_ part of the fix! Read my explanation above, otherwise everything gets stuck in idle thread. I could make a separate PR for this, but its already a commit and as said its part of the fix.